README.md

```diff
@@ -1,15 +1,15 @@
-# Mondoo Chef Cookbook
+# Mondoo Package Chef Cookbook
 
 [![Cookbook Version](https://img.shields.io/cookbook/v/mondoo.svg)](https://supermarket.chef.io/cookbooks/mondoo)
 [![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 
-This cookbook installs the Mondoo Client on Linux servers.
+This cookbook installs the Mondoo `cnquery` and `cnspec` on Linux servers.
 
 It does:
 
 * Installs the signed `mondoo` package
-* Registers the client with Mondoo Platform
-* Enables the systemd service
+* Login `cnquery` and `cnspec` with Mondoo Platform
```

Comment:
Login is confusing in this wording

Reply:
Any other wording is recommended

Reply:
It should just be register. Registers works. Logins does not.

Reply:
The command has been renamed to login and register command does not exist anymore. I am not sure it makes sense to confuse users with an old term then.

Reply:
It's confusing text, but it's not the end of the world.